INSTALL

```diff
@@ -163,64 +225,64 @@ You can run the following command:
 sudo ldconfig ~/src/install-root/lib
 
 
-Building Subsurface under MacOSX (using Homebrew)
--------------------------------------------------
+Building Subsurface under MacOSX
+--------------------------------
 
-0) You need to have XCode installed. The first time (and possibly after updating OSX)
-you need to run
+Simple version:
 
-$ xcode-select --install
+brew install libzip cmake libusb pkg-config automake libtool hidapi curl libssh2
 
-1)  Install Homebrew
 
-$ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+0) You need to have XCode installed. The first time (and possibly after updating OSX)
 
-2) Install needed dependencies
+0.1) run "xcode-select --install"
 
-$ brew install asciidoc libzip cmake libusb pkg-config automake libtool hidapi curl libssh2
+1) You also need to install pkg-config
 
-3) Download and install Qt
+1.1) download http://sourceforge.net/projects/macpkg/files/PkgConfig/0.26/PkgConfig.pkg/download
 
-You can build Qt from source or use the prebuilt binaries for Mac. Start
-by downloading the online installer:
+1.2) run "PkgConfig.pkg"
 
-$ curl -L -o ~/Downloads/qt-unified-mac-x64-online.dmg \
-	http://download.qt.io/official_releases/online_installers/qt-unified-mac-x64-online.dmg
-$ open ~/Downloads/qt-unified-mac-x64-online.dmg
+1.3) run "sudo ln -s /opt/pkgconfig/bin/pkg-config /usr/local/bin/pkg-config"
 
-Double click on the Qt installer shown in the Finder window.
+2) install autoconf tools
 
-In the installer, chose an install folder (the build script we are using
-below assumes that you accept the default of /home/<your username>/Qt), in
-"Select components" select the most recent version and (if you want to
-build Qt from source) be sure you also install the "Source Components".
-To save time and disk space you can unselect Android and IOS packages
-as well as QtWebEngine, Qt3D, Qt Canvas 3D and the Qt Extras.
+2.1) curl -OL http://ftpmirror.gnu.org/autoconf/autoconf-latest.tar.gz
 
-If you want to build from source (which takes a very long time and a lot of disk)
+2.2) tar -xzf autoconf-latest.tar.gz
 
-$ cd ~/Qt/5.9.2/Src/
+2.3) cd autoconf-*
 
-$ ./configure -prefix /usr/local -opensource
+2.4) ./configure && make && sudo make install
 
-$ make -j4
+2.5) cd ..
 
-$ make install
+3) install 
 
-5) run the build script
+4) run the build script
 
-cd ~/src
-bash subsurface/scripts/build.sh
+4.1) cd <repo>/..
+
+4.2) run "bash <repo>/scripts/build.sh -build-deps -desktop
 
 After the above is done, Subsurface.app will be available in the
 subsurface/build directory. You can run Subsurface with the command
 
-$ open subsurface/build/Subsurface.app
+4) open subsurface/build/Subsurface.app
```

Comment:
The disadvantage of running it like this is that you won't see diagnostic messages in the terminal.

To get those you directly run

subsurface/build/Subsurface.app/Contents/MacOS/Subsurface

(the TAB key is your friend here).

Maybe we should also mention that using Qt Creator (with our style settings from the coding style document) is the suggested way for development (thanks to commend completion, following symbols around and having a GUI for the debugger).

Reply:
Actually that is the old text, but I will update with your explanation.